docs/usage.md

```diff
@@ -221,48 +221,26 @@ For example:
 ```bash
 --split_fastq 10000
 ```
+### --trim_fastq
+Use this to perform adapter trimming [Trim Galore](https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md)
 
-### --sample
```

Comment:
Why removing part of the current documentation?

Reply:
sorry error, let me fix that